acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  BowlOrderDetailViewController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+class BowlOrderDetailViewController: UIViewController {
+    
+    // MARK: View vars
+    
+    var collectionView: UICollectionView!
+    var collectionViewListAdapter: ListAdapter!
+    
+    // MARK: Data
+    
+    var bowlItem: BowlItem!
+    var baseOptions: [CustomizationOption]!
+    var toppingOptions: [CustomizationOption]!
+    
+    // MARK: View Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
you don't need `translatesAutoresizingMaskIntoConstraints = false` when we use `SnapKit` so delete all instances of these